extensions/ql-vscode/package.json

```diff
@@ -340,6 +340,12 @@
           "type": "boolean",
           "default": false,
           "description": "Allow database to be downloaded via HTTP. Warning: enabling this option will allow downloading from insecure servers."
+        },
+        "codeQL.createQuery.folder": {
+          "type": "string",
+          "default": "",
+          "patternErrorMessage": "Please enter a valid folder",
+          "markdownDescription": "The name of the folder where we want to create queries and query packs via the \"CodeQL: Create Query\" command. The folder should exist in the workspace."
```

Comment:
Minor observation:
> The folder should exist in the workspace.

We don't actually enforce that the folder should be in the workspace (it just needs to exist _somewhere_). E.g. 
- I'm in workspace `foo` and I set `codeQL.createQuery.folder` to "foo/my-folder"
- If I change to a different workspace `bar` and run "Create Query", the query still gets created in "foo/my-folder"

It's a bit odd, but probably fine as-is. Users can always manually override `codeQL.createQuery.folder` in their [workspace-specific settings](https://code.visualstudio.com/docs/getstarted/settings#_workspace-settings) 🤷🏽

Reply:
Right! Ok, I can drop the "in the workspace" part. Thanks Shati! 😍